lib/liquid/standardfilters.rb

```diff
@@ -194,8 +194,12 @@ def map(input, property)
         if property == "to_liquid".freeze
           e
         elsif e.respond_to?(:[])
-          r = e[property]
-          r.is_a?(Proc) ? r.call : r
+          begin
+            r = e[property]
+            r.is_a?(Proc) ? r.call : r
```

Comment:
This line should not be part of the `begin` block, or we may catch other errors unrelated to the `e[property]` method call.